schema/rdl_schema_0.1.json

```
@@ -1025,6 +1025,37 @@
 				"Judgement",
 				"Hybrid"
 			]
+		},
+		"Temporal": {
+			"title": "Temporal coverage",
```

Like in https://github.com/GFDRR/rdl-standard/pull/105#discussion_r1244481070, I realised that the definition should actually be called 'Period'. The changelog and `$ref`s will need updating, too.

```suggestion
		"Period": {
			"title": "Period",
```